catalog/ui/catalog-ui-search/src/main/webapp/component/select/select.view.js

```diff
@@ -41,11 +41,21 @@ define([
                 'data-help': this.model.get('help')
             };
         },
+        onRender: function(){
+            if (this.model.get('description')) {
+                this.$el.attr('data-help', this.model.get('description'));
```

Comment:
Or is there just no tooltip if there's no entry in the AttributeDescriptions map?

Reply:
Aye. The `if` will be false for an `undefined` value and no `data-help` atribute will be set. This particular line sets the context-sensitive help, incidentally (when you press the ? button). Tooltips are set in this view's template (the HTML `title` attribute).